app/views/overview/_service-instance-row.html

```diff
@@ -0,0 +1,133 @@
+<div class="list-pf-item" ng-class="{ active: row.expanded }">
+  <div class="list-pf-container" ng-click="row.toggleExpand($event)">
+
+    <div class="list-pf-chevron">
+      <div ng-include src=" 'views/overview/_list-row-chevron.html' " class="list-pf-content"></div>
+    </div>
+
+    <div class="list-pf-content">
+      <div class="list-pf-name">
+        <h3>
+          <a ng-href="{{row.apiObject | navigateResourceURL}}">
+            <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
+          </a>
+          <div class="component-label" style="padding-top: 4px;">
```

Comment:
get rid of this inline style, if 5px will look ok then use the pad-top-sm, if not move this to the CSS